v3/setup/splunk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# SPLUNK_FORWARD_SERVER shoudl be delcared as a config in the setup-credentials util
+
+source /etc/environment
+
+etcdctl set /splunk/SPLUNK_FORWARD_SERVER $SPLUNK_FORWARD_SERVER
```

Comment:
Where is `$SPLUNK_FORWARD_SERVER` coming from? I'm assuming it is an environment variable, but did you update infrastructure to set it in user data? Or is it set somewhere else?